### PR TITLE
[Feature] Added hunter-killer ACE action to crew slots of all land and sea vehicles.

### LINKED ISSUE #142: [Feature Request] Add support for ACE hunterkiller to the logi vehicle
**Is your feature request related to a problem? Please describe.**
Vehicles spawned from the logi system do not have ACE_hunterkiller enabled by default (except vanilla)

**Describe the solution you'd like**
Tell logi system when you want to have the system enabled on spawned tanks. This requires applying all brain wrinkles to figure out how Bubbus' macros work and then finding an elegant solution to add Joe's current extremely bad solution.

**Why is this this best way to solve the problem?**
idk, probably a better way somewhere

### PR DESCRIPTION
**Pull request by Bubbus**
Resolves #142.

### Pull Request Description
**When merged this pull request will:**
- Add hunter-killer ACE action to crew slots of all land and sea vehicles.

### Release Notes
Added hunter-killer ACE action to crew slots.  You can now turn on ACE Hunter-Killer mode when you're crewing a vehicle and use it to point out targets for your gunner.  This action is available in all vehicles that AI Driver is available for.

## IMPORTANT

- [x] Testing has been completed as neccessary, depending on the nature & impact of the changes.
- [x] The Release Notes section below must be filled out appropriately to explain the changes made in this PR. This section will be used in Framework Changelogs.
- [ ] If the contribution affects [the wiki](https://github.com/CombinedArmsGaming/CAFE3/wiki), please include your changes in this pull request so the wiki is consistently updated.
- [x] [Contribution Guidelines](https://github.com/CombinedArmsGaming/CAFE3/blob/release/contributing.md) are read, understood and applied.
- [x] Title of this PR uses our standard template `[Descriptor] - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: components/aceActions/fn_addHunterKillerActions.sqf
Comment (Cre8or) [inline components/aceActions/fn_addHunterKillerActions.sqf:1]: It took me a minute to figure out that the actions don't have anything to do with the driver role/unit. I'm guessing you copied this from the AI driver action? If so, please rename `_driverNotActive` and `_driverActive` to something else (maybe `_isHunterKillerActive` / `_isHunterKillerInactive`?).

File: startup/components/globals/serverGlobals.sqf
Comment (Cre8or) [inline startup/components/globals/serverGlobals.sqf:1]: Good catch!

**Review by Cre8or — CHANGES_REQUESTED** [review]
Yet another solid PR - we're approaching pedantic levels of reviewing because everything else is already spot on. 👍 

Thank you!

File: components/aceActions/fn_addHunterKillerActions.sqf
Comment (Bubbus) [reply components/aceActions/fn_addHunterKillerActions.sqf:1]: oops

**Review by Cre8or — APPROVED** [review]
🚢

**Pull request merged**